packages/commands-http/src/request.ts

```diff
@@ -0,0 +1,84 @@
+import { Command } from '@seagull/commands'
+import fetch, { Response as FetchResponse } from 'node-fetch'
+import { Http, RequestConfig } from './index'
```

Comment:
Circular dependencies are not a good idea. Can we somehow prevent that?

Reply:
Valid point, not needed at this point, removed